packages/site/cms-site/src/vendor.d.ts

```diff
@@ -1 +1,9 @@
 /// <reference types="next" />
+
+import "react";
+
+declare module "react" {
+    interface CSSProperties {
+        [key: `--${string}`]: string | number;
+    }
+}
```

Comment:
This is necessary to avoid a type error when setting css variables via the `style` prop